app/src/main/java/space/taran/arknavigator/utils/LogTags.kt

```diff
@@ -38,3 +38,5 @@ const val IMAGES: String = "images"
 const val GLIDE: String = "glide"
 
 const val FILES: String = "files"
+
+const val KIND: String = "Kind: "
```

Comment:
Should be somewhere else. This file is called `LogTags` and used only for module names used with logging.

Reply:
updated in new PR